Template.php

```diff
@@ -558,7 +558,8 @@ public function add_if_new($param_name, $value, $api = NULL) {
       case "page": case "pages":
         if (( $this->blank("pages") && $this->blank("page") && $this->blank("pp")  && $this->blank("p") && $this->blank('at'))
                 || strpos(strtolower($this->get('pages') . $this->get('page')), 'no') !== FALSE
-                || (strpos($value, chr(2013)) || (strpos($value, '-'))
+                || (strpos($value, chr(2013)) || (strpos($value, '-')) || (strpos($value, REGEXP_EN_DASH))
```

Comment:
Is `REGEXP_EN_DASH` compatible with `strpos`?

Reply:
Shocking yes.  I actually had to test it out.  Good question